src/sentry/static/sentry/app/views/sharedGroupDetails/index.jsx

```diff
@@ -95,7 +95,12 @@ const SharedGroupDetails = React.createClass({
             <div className="content">
               <SharedGroupHeader group={group} />
               <div className="group-overview">
-                <EventEntries group={group} event={evt} isShare={true} />
+                <EventEntries
+                  group={group}
+                  event={evt}
+                  orgId={group.project.organization.slug}
```

Comment:
we should try to get some tests in for this (i might take a stab at it) as this was missing required props